client/lib/themes/theme_constants.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+Color primary = const Color(0xFFFF8714);
+Color onprimary = const Color(0xFF000000);
+Color secondary = const Color(0xFFF5F5F5);
+Color onsecondary = const Color(0xFFFFFFFF);
+Color background = const Color(0xFFFFFFFF);
+const apppadding = 16.0;
```

Comment:
Please add the type + add an uppercase => appPadding

Reply:
??